src/main/sensors/gyro.c

```diff
@@ -189,14 +190,15 @@ PG_RESET_TEMPLATE(gyroConfig_t, gyroConfig,
     .gyro_soft_notch_cutoff_1 = 0,
     .gyro_soft_notch_hz_2 = 0,
     .gyro_soft_notch_cutoff_2 = 0,
-    .checkOverflow = GYRO_OVERFLOW_CHECK_ALL_AXES,
+    .checkOverflow = GYRO_OVERFLOW_CHECK_YAW,
     .imuf_mode = 32,
+    .imuf_rate = IMUF_RATE_32K,
     .imuf_pitch_q = HELIO_PROFILE_PITCH_Q,
-    .imuf_pitch_r = 88,
+    .imuf_pitch_w = HELIO_PROFILE_PITCH_W,
```

Comment:
These should not take from a specific target for future target compatibility

Reply:
I will change it to IMUF and ensure it's defined in the driver header file